examples/social/tests/cross_chain.rs

```diff
@@ -48,14 +48,24 @@ async fn test_cross_chain_posting() {
     chain1.handle_received_effects().await;
 
     // Querying the own posts
-    let query_string = "query { ownPosts(start: 0, end:1) { timestamp, text } }";
-    let value: serde_json::Value = chain2.query(application_id, query_string).await;
-    let value = value["data"]["ownPosts"][0]["text"].clone();
+    let query_string = "{ ownPosts(start: 0, end:1) { timestamp, text } }";
```

Comment:
I suspect we can safely replace `query_string` by `query` just about everywhere.